install/intro.xml

```diff
@@ -0,0 +1,91 @@
+    </itemizedlist>
+   </para>
+   <para>
+    Per la prima e più comune forma, sono necessarie tre cose: 
+    PHP, un server web e un browser web, probabilmente avrete già un 
+    web browser, e, conformemente alle impostazioni del vostro 
+    sistema operativo, potreste avere anche un server web (es. 
+    Apache su Linux e macOS o IIS su Windows). Potete anche aver 
+    noleggiato dello spazio web da un fornitore. In questo modo 
+    non avrete bisogno di impostare niente da parte vostra, solo 
+    scrivere i vostri script PHP, caricarli sul server che avete 
+    affittato e vedere i risultati sul vostro browser. 
+   </para>
+   <para>
+    Nel caso in cui vogliate impostare il server e PHP da voi stessi, 
+    avete due possibilità di scelta per il modo in cui collegare PHP 
+    al vostro server. Per molti server PHP ha un'interfaccia modulare 
+    diretta (altrimenti chiamata SAPI). Tra questi server sono inclusi 
+    Apache, Microsoft Internet Information Server, Netscape e iPlanet. 
+    Se PHP non ha i moduli per supportare il vostro web server è sempre 
+    possibile usarlo come processore CGI o fast CGI. Questo significa 
+    che è possibile impostare il prprio server per usare il codice 
+    eseguibile CGI di PHP per processare tutte le richieste PHP di file 
+    fatte al server. 
+   </para>
+   <para>
+    Se si è anche interessati all'uso di PHP per la scrittura di script 
+    per la riga di comando (cioè scrivere script per la generazione di 
+    alcune immagini offline, o per processare file di testo in base a 
+    degli argomenti che possiamo impostare), avrete sempre bisogno di 
+    una riga di comando eseguibile. Per ulteriori informazioni, leggere 
+    la sezione sulla <link linkend="features.commandline">scrittura di 
+    applicazioni PHP per la riga di comando</link>. In questo caso non 
+    è necessario un server e neanche un browser. 
+   </para>
+   <para>
+    Con PHP è anche possibile scrivere applicazioni desktop con interfaccia 
+    grafica usando l'estensione PHP-GTK. Questo è un approccio completamente 
+    diverso dalla scrittura di pagine web, voi non genererete alcun HTML, ma, 
+    dovrete maneggiare finestre e oggetti all'interno di esse. Per ulteriori 
+    informazioni su PHP-GTK 
+    <link xlink:href="&url.php.gtk;">visitate il sito dedicato a questa estensione
+    </link>. PHP-GTK non è incluso nelle distribuzioni ufficiali di PHP. 
+   </para>
+   <para>
+    Da qui in avanti, questa sezione proseguirà con l'impostazione di PHP 
+    per i web server su Unix e Windows con moduli e interfacce per server 
+    ed eseguibili CGI. Si possono anche trovare informazioni sugli eseguibili 
+    per riga di comando alle seguenti sezioni. 
+   </para>
+   <para>
+    Il codice nativo e le distribuzioni binarie di PHP per Windows possono 
+    essere trovate a <link xlink:href="&url.php.downloads;">&url.php.downloads;</link>.
```

Comment:
Questa sezione ha un rigo in meno rispetto alla versione inglese. Andrebbe sistemato.

Reply:
Questo problema è ancora presente.